src/containers/BooksList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import Book from '../components/Book';
+
+export default function BooksList() {
```

Comment:
- [ ] The `book object` should be passed as a prop and passed to each `Book` component